### ISSUE TITLE
Problems with vagrant-spk dev

### ISSUE THREAD

**Issue by Moredread** [jparyani/mediawiki-sandstorm#5]
Hey,

I tried to run the dev version of mediawiki with vagrant-spk.

First, there seems to be an issue with missing escapes for '$' in the setup.sh. I've send a PR, #4, but when I run vagrant-spk and create a grain, it complains that the LocalSettings.php for mediawiki is missing.

The log reads

```
...** SANDSTORM SUPERVISOR: Starting up grain.
A copy of your installation's LocalSettings.php
must exist and be readable in the source directory.
Use --conf to specify it.
waiting for php5-fpm to be available at /var/run/php5-fpm.sock
```

Any idea where this issue comes from? Also the app graphic is missing at the grain creation page, and the app has the name 'Example App'.

**Comment by jparyani** [comment]
I'm not sure exactly what problem you're running into.

The "Example App"/no icon thing makes me think sandstorm-pkgdef.capnp got replaced. Did you run `vagrant-spk init` by accident? Make sure .sandstorm/sandstorm-pkgdef.capnp has no changes in it.

**Comment by Moredread** [comment]
That might have happend. I checked out the current version again, and it works now. :) I get errors due to the missing pgp-keyring file. If I comment it out in the capnp file, I can get it to run. Is there a way to skip the pgp signature during dev?

**Comment by jparyani** [comment]
Sorry, I had forgotten to checkin the pgp files. Fixed in 5112e6958d87342fa518b6a91e02b1e092b07796
